ui/components/multichain/account-list-menu/account-list-menu.js

```diff
@@ -85,24 +87,22 @@ export const AccountListMenu = ({ onClose }) => {
       <ModalOverlay />
       <ModalContent
         className="multichain-account-menu-popover"
-        modalDialogProps={{ padding: 0, marginBottom: 0 }}
+        modalDialogProps={{
+          className: 'multichain-account-menu-popover__dialog',
+          padding: 0,
+          display: Display.Flex,
+          flexDirection: FlexDirection.Column,
```

Comment:
By employing `display: flex` on the parent element and setting `overflow: scroll` on the child element, we can achieve dynamic resizing of the ModalContent's dialog without causing it to exceed the available screen space limit.